opentracing/src/main/java/io/opentracing/Tracer.java

```diff
@@ -87,15 +87,18 @@
       /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
       SpanBuilder withTag(String key, String value);
 
-      /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
+      /** Same as {@link Span#setTag(String, boolean)}, but for the span being built. */
       SpanBuilder withTag(String key, boolean value);
 
-      /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
+      /** Same as {@link Span#setTag(String, Number)}, but for the span being built. */
       SpanBuilder withTag(String key, Number value);
 
-      /** Specify a timestamp of when the Span was started, represented in microseconds since epoch. */
+      /** Same as {@link Span#setBaggageItem(String, String)}, but for the span being built. */
       SpanBuilder withStartTimestamp(long microseconds);
 
+      /** Specify a timestamp of when the Span was started, represented in microseconds since epoch. */
+      SpanBuilder withBaggage(String key, String value);
```

Comment:
It would be `withBaggageItem(..)`.

Do we need this as part of the builder? The baggage state will never influence the creation.